ledger/participant-integration-api/src/main/scala/platform/indexer/JdbcIndexer.scala

```diff
@@ -196,11 +199,15 @@ object JdbcIndexer {
       }
     }
 
-    Source
-      .futureSource(lfPackagesSource())
-      .mapAsyncUnordered(config.initLoadParallelism)(loadLfArchive)
-      .mapAsyncUnordered(config.initProcessParallelism)(processPackage)
-      .runWith(Sink.foreach(packageMetadataView.update))
+    Timed
+      .future(
+        metrics.daml.index.packageMetadata.viewInitialisation,
```

Comment:
Isn't this the same duration computed on line 218 (wrt to deadline exceeded). Can we deduplicate these?